products/rancher.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Rancher
+category: service
+tags: suse rancher kubernetes
```

Comment:
```suggestion
tags: suse
```

We try to only add tags that have chances to be reused, with the exception of company names developing the product. So this only leave the `suse` tag which is the company behind rancher.